src/components/App.js

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import Button from './Button'
+import Header from './Header'
+import Prompt from './Prompt'
+
+function App() {
+  const prompts = [
```

Comment:
maybe you can load this data from an different file, so the code will get a bit shorter :)